simulation/model.py

```diff
@@ -29,11 +30,13 @@ def __init__(
         household_num_lookahead_years: int,
         heating_system_hassle_factor: float,
         interventions: Optional[List[InterventionType]],
-        air_source_heat_pump_discount_factor_2022: float,
         gas_oil_boiler_ban_datetime: datetime.datetime,
         price_gbp_per_kwh_gas: float,
         price_gbp_per_kwh_electricity: float,
         price_gbp_per_kwh_oil: float,
+        air_source_heat_pump_price_discount_schedule: List[
+            Tuple[datetime.datetime, float]
+        ],
```

Comment:
Can we change this to an `Optional[List[...]]` to avoid passing in `[]` to instantiate it?
```suggestion
        air_source_heat_pump_price_discount_schedule: Optional[List[
            Tuple[datetime.datetime, float]
        ]],
```